### ISSUE TITLE
Extend visibility-first to also check constants

### ISSUE THREAD

**Issue by sohkai** [duaraghav8/Solium#239]
Constant definitions should also be checked by `visibility-first` (or an alternative rule) so that their visibility modifiers come first.

**Comment by duaraghav8** [comment]
I agree that `visibility-first` itself should check constants too.

But can you tell me how this is beneficial?
Is your use case simply that it is a good practice to enforce visibility on constants too? (If that is the case, then the same may apply to non-const state vars too, although I'm not sure)

**Comment by sohkai** [comment]
> It is a good practice to enforce visibility on constants

I would agree with this, and in general explicitly stating visibility is a good thing (maybe there should be another rule for `visibility-always`?).

But when stating visibility, it'd be nice if all state and methods had their visibility in the same order (our code base had gotten out of sync where our constants had their visibility defined later).
